MAINTAINERS.md

```diff
@@ -0,0 +1,26 @@
+# Maintainers
+
+This document is for project maintainers who are doing versioning, releases, and publishing for the cardstack mono-repo.
+
+## Updating package versions
+
+The versions of all packages in the mono-repo are updated at the same time, to the same version number.
+
+1. Get the latest code on master and create a branch. `git checkout master`, `git pull origin master`, `git checkout -b vX.Y.Z`
+2. Look at the `git log` since the last release, and list anything significant in the `CHANGELOG.md`.
+Determine whether the release should be a major (breaking) or minor (non-breaking).
+For breaking releases, include upgrade instructions in the changelog.
+3. Commit the changes, make sure your working tree is clean, and push your branch to GitHub.
+4. To update versions in the `package.json` for all packages, use `npx lerna version --force-publish`. `npx` will use the version of `lerna` that is installed as a `devDependency` of the mono-repo. `--force-publish` will update interdependencies as well as the version numbers of the packages themselves.
```

Comment:
I think you actually want `--force-publish="*"`, at least that is what I've been using. We've also been using `--exact` instead of allowing a range (i.e. `^`) to be used.

Reply:
should we actually just use `lerna publish` instead of `lerna version`? that's what we've done in the past. It was a bit more yolo, but it was just a single command to do the versioning and publishing. I'm not necessarily opposed to opening PR's for releasing the hub. Maybe @ef4 has thoughts here.

Reply:
Changed to `npx lerna version --force-publish="*" --exact`

Reply:
I think lerna publish would be ok too. It just has fewer guardrails. Eventually I'd like to write something that checks that you have the latest commits on master before publishing. Then it would be safer to do in one command.

Reply:
I added a section with this command:

```sh
npx lerna publish from-git --force-publish="*" --exact
```

@habdelra does that seem correct?